cf/samples/javascript-samples/sap-iots-device-rest/index.js

```diff
@@ -0,0 +1,163 @@
+/** 
+ * Code sample for demonstrating Cloud-to-Cloud integration with SAP IoT Service.
+ *
+ * Proxy objects instantiated from this module allow for POSTing data to SAP IoT Service Cloud.
+ * The data is posted on behalf of the IoT device, the credentials of which are provided while
+ * creating the proxy object.
+ * 
+ * The module uses asynchronous functions (co-routines) extensively. Several HTTP connections
+ * can be established and used in parallel by a single proxy object.
+ *
+ * @module sap-ios-device-rest
+ * @author Mikhail Bessonov <mikhail.bessonov@sap.com>
+ * @license SAP Sample Code License Agreement
```

Comment:
Same here, is it possible to link it? should we do it?

Reply:
Added a plain text reference to the LICENSE file.